src/common/mime_types.rs

```diff
@@ -29,6 +29,10 @@ impl TryFrom<&str> for MimeType {
 
     fn try_from(arg: &str) -> Result<Self> {
         if let Ok(url) = url::Url::parse(arg) {
+            if url.scheme() == "file" {
+                print!("{}", url.path());
```

Comment:
The print here seems like a debugging leftover ?

Reply:
Yes, it was, sorry.

Reply:
Any update? without this, I can't use `handlr` with telegram-desktop.